app/generated_resources.grd

```diff
@@ -10863,6 +10863,9 @@ Please help our engineers fix this problem. Tell us what happened right before y
     <message name="IDS_SHOW_KALEIDOSCOPE" desc="Menu item to launch the Brave watch feature">
       Watch
     </message>
+    <message name="IDS_EXTENSION_PROMPT_WARNING_IPFS" desc="Permisson string for access to ipfs api.">
+      Publish and retrieve information IPFS files
```

Comment:
This sounds a bit confusing. `IPFS information files`? @bbondy

Reply:
Sorry @spylogsster one more attempt here, how about this?
Publish and retrieve files from IPFS

Reply:
updated